docs/sampling.md

```diff
@@ -20,29 +20,26 @@ Tracer tracer = wfTracerBuilder.build();
 ```
 ## Supported Sampling Strategies
 
-The following table lists the supported sampling strategies. You create and configure a sampling strategy by configuring an `Sampler` implementation:
+The following table lists the supported sampling strategies. You create and configure a sampling strategy by configuring a `Sampler` implementation:
 
 | Sampler              | Description                            |
 | --------------------- | -------------------------------------- |
 | ConstantSampler       | Allows either all traces or no traces. Specify `true` to sample all traces, or `false` to sample no traces. |
 | DurationSampler       | Allows a span if its duration exceeds a specified threshold. Specify the duration threshold as a number of milliseconds. |
 | RateSampler           | Allows a specified probabilistic rate of traces to be reported. Specify the rate of allowed traces as a number between 0.0 and 1.0. |
-| CompositeSampler      | Delegates the sampling decision to multiple other samplers and allows a span if any delegate decides to allows it. Specify a list of samplers to delegate to. |
+| CompositeSampler      | Delegates the sampling decision to multiple other samplers and allows a span if any delegate decides to allows it. Specify a list of samplers to delegate. |
```

Comment:
I think the original text, Specify a list of samplers to delegate to, is more precise. I delegate to the samples, no?